src/InsertionsClient/ConsoleApp/Program.cs

```diff
@@ -51,6 +53,8 @@ internal class Program
 
         private static string IgnoredPackagesFile = string.Empty;
 
+        private static bool IgnoreDevUxTeamPackagesCase;

```

Comment:
nit: The `Case` suffix makes it look like it has something to do with case-insensitive comparison.

Reply:
changed to `IgnoreDevUxTeamPackagesScenario`